pkg/console/starter/starter.go

```diff
@@ -152,6 +154,15 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		resourceSyncer,
 	)
 
+	service.NewServiceSyncController(
+		operatorClient,
+		kubeClient,
+		kubeInformersNamespaced,
+		api.OpenShiftConsoleNamespace,
```

Comment:
Passing in names to our controllers should make them a bit more flexible.

Reply:
yeah I would rather remove this since the value will be always the same. Not sure why would we reuse this one ?

Reply:
I think its good practice for controller patterns, perhaps we will have another operator in the future and would want to copy a controller across operators but not have to refactor a bunch of hard-coded values.

Reply:
ok :)